example/mpi.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: scheduling.incubator.k8s.io/v1alpha1
+kind: MPI
+metadata:
+  name: test-mpi
+spec:
+  job:
+    apiVersion: batch/v1
+    kind: Job
+    spec:
+      completions: 6
+      parallelism: 6
+      template:
+        spec:
+          containers:
+          - image: continuse/mpich:v3
+            imagePullPolicy: IfNotPresent
+            name: mpi
+            command: ["mpirun", "-n", "6", "hostname"]
+            resources:
+              requests:
+                cpu: "200m"
+  podGroup:
+    spec:
+      minMember: 6
```

Comment:
Can this member come from job/parallelism or deployment/replicas directly? As a user, my expectation is controller create podGroup for my job automatically.

Reply:
Absolutely, I'll improve that.

Reply:
Done :)